### DISCUSSION TITLE
Multi language in Builder

### DISCUSSION THREAD

**Discussion by SOVLOOKUP** [Budibase/budibase#1150] [Ideas]
Any mutil language plan before official release? 
[i18next](https://github.com/i18next/i18next) seems like a nice solution  #35 
It's faster to use the tool [i18next/i18next-scanner](https://github.com/i18next/i18next-scanner) (If it can support svelte)

**Comment by joebudi** [comment]
Great suggestion. We have spoken about this internally. Once the platform's UI is more stable, and we are out of beta, we will strive to  support multiple languages. Right now, there are too many moving parts.

**Reply by Bogey** [reply]
@joebudi Until something formal is ready, how do you suggest to approach a multi-language project?

**Reply by mjashanks** [reply]
@Bogey Unfortunately there is nothing that you can do in the meantime. Of course, this only applies in certain areas - https://github.com/Budibase/budibase/discussions/2023 does a great job of listing some of those.

Just to clarify - when you say "multi-language project" - do you mean that you are trying to build a single app in multiple languages?

Thanks.

**Reply by Bogey** [reply]
> do you mean that you are trying to build a single app in multiple languages

Yes.
I could of course deploy three apps, the same app but in a different languages. 
However, users cannot switch from one language to another without losing their current path (I guess or is there a possibility for htaccess rewrites?).

Thanks :)
